perses/tests/test_relative_setup.py

```diff
@@ -260,47 +258,6 @@ def warn(*args, **kwargs):
                 found_tla = True
         assert found_tla == True, 'Spectator TLA not in old topology'
 
-
-def test_host_guest_deterministic_geometries():
```

Comment:
Why is this test being deleted?

Reply:
It basically got replaced by the test at https://github.com/choderalab/perses/blob/499b39ea3b61d4b21377d65343bd7afbe10517bc/perses/tests/test_relative_setup.py#L368

The diff between the two is shown in the following screenshot 
![perses_tests_diff](https://github.com/choderalab/perses/assets/2949729/41ef68bf-5d0e-412f-8b03-1e068fd2ef5a)

You can see only the forcefield was changed between the tests (since we want to be testing openff rather than gaff, preferably), and the lines to test the storage pdbs were added.